runtime/onert/frontend/base_loader/include/base_loader.h

```diff
@@ -361,7 +364,18 @@ ir::OperandIndex BaseLoader<LoaderDomain, SpecificLoader>::loadOperand(const Ten
   const auto *data = _model->buffers()->Get(tensor->buffer())->data();
   if (data != nullptr)
   {
-    auto ptr = std::make_unique<ir::MMapedData>(_base, _pagesize, data->data(), data->size());
+    using std::ptrdiff_t;
+    size_t data_size = data->size();
+    ptrdiff_t unaligned_offset_start = data->data() - _base;
+    ptrdiff_t offset_end = unaligned_offset_start + data_size;
+
+    // Calculated aligned offset from base address of mapped region
+    // munmap accepts memory address which is a multiple of the pagesize
+    ptrdiff_t aligned_offset_start = (unaligned_offset_start / _pagesize) * _pagesize;
+    size_t mmap_size = offset_end - aligned_offset_start;
+
+    auto ptr = std::make_unique<ir::MMapedData>(_fd, aligned_offset_start, mmap_size,
```

Comment:
Is this eventually mmaped twice? Is it your intention?

Reply:
Yes. mmaped twice (full model file loading, constant operand data), and unmap 
- full file loading: munmap after model file loading (model graph generation) finish
- constant operand data: munmap on `MMapedData` destructor